wwwapp/auth.py

```diff
@@ -1,37 +1,33 @@
-#-*- coding: utf-8 -*-
-from __future__ import unicode_literals
-
 import base64
 import hashlib
 import re
 
+from allaccess.models import AccountAccess
+from allaccess.views import OAuthRedirect, OAuthCallback
 from django.conf import settings
 from django.contrib.auth import authenticate, login
 from django.contrib.auth.models import Group, User
 from django.core.exceptions import ValidationError
-from django.core.urlresolvers import reverse
-from django.dispatch import receiver
 from django.db.models.signals import post_save
+from django.dispatch import receiver
 from django.shortcuts import render, redirect
+from django.urls import reverse
+from django.utils.encoding import smart_bytes, force_text
 
-from allaccess.compat import smart_bytes, force_text
-from allaccess.models import Provider, AccountAccess
-from allaccess.views import OAuthRedirect, OAuthCallback
+from .models import UserProfile, UserInfo
+from .views import get_context
 
-from models import UserProfile, UserInfo
-from views import get_context
 
-
-def loginView(request):
+def login_view(request):
     context = get_context(request)
-    
+
     # Forget AccountAccesses to merge if user goes somewhere then back to login.
     if 'merge_access' in request.session:
         del request.session['merge_access']
     if 'merge_access_info' in request.session:
         del request.session['merge_access_info']
-    
-    if request.user.is_authenticated():
+
+    if request.user.is_authenticated:
```

Comment:
Is this correct? (I'm not sure it changed in Django)

Reply:
Using it with `()` as a method rather than an attribute was [deprecated in 1.10](https://docs.djangoproject.com/en/2.2/releases/1.10/#using-user-is-authenticated-and-user-is-anonymous-as-methods) and [completely removed in 2.0](https://docs.djangoproject.com/en/2.2/releases/2.0/#features-removed-in-2-0)